### PR TITLE
elasticsearch 6.0.0 and elasticsearch@5.6 5.6.4 (new formula)

### PR DESCRIPTION
**Pull request by jasontedor**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

**Comment by jasontedor** [comment]
Please note the elasticsearch@5.6 formula in this PR is simply a rename of the existing formula to 5.6; as that formula is already accepted into this repository, elasticsearch@5.6 should not need a review here.

**Review by ilovezfs — CHANGES_REQUESTED** [review]
Please make the addition of elasticsearch@5.6 a separate commit with the commit subject `elasticsearch@5.6 5.6.4 (new formula)`

File: Formula/elasticsearch.rb [hunk 1]
```diff
@@ -40,46 +40,33 @@ def install
...
-    end
-
     # Move config files into etc
     (etc/"elasticsearch").install Dir[libexec/"config/*"]
-    (etc/"elasticsearch/scripts").mkdir unless File.exist?(etc/"elasticsearch/scripts")
     (libexec/"config").rmtree
 
-    bin.write_exec_script Dir[libexec/"bin/elasticsearch"]
-    bin.write_exec_script Dir[libexec/"bin/elasticsearch-plugin"]
+    bin.install libexec/"bin/elasticsearch", libexec/"bin/elasticsearch-keystore", libexec/"bin/elasticsearch-plugin", libexec/"bin/elasticsearch-translog"
```
Comment (ilovezfs) [inline Formula/elasticsearch.rb]: would be good to avoid going passed the eightieth column

**Comment by ilovezfs** [comment]
The versioned formula needs to be `keg_only :versioned_formula`

**Comment by jasontedor** [comment]
This is split into two commits now and I have made the other requested changes.

**Comment by ilovezfs** [comment]
Thanks @jasontedor! :balloon:

**Comment by jasontedor** [comment]
Thanks @ilovezfs!

**Pull request merged**